test/passes/Oz_fuzz-exec_all-features.txt

```diff
@@ -530,10 +429,6 @@
 [LoggingExternalInterface logging 99]
 [LoggingExternalInterface logging 0]
 [LoggingExternalInterface logging 10]
-[fuzz-exec] calling rtt_Fresh
-[LoggingExternalInterface logging 1]
-[LoggingExternalInterface logging 0]
-[LoggingExternalInterface logging 1]
 [fuzz-exec] calling array.init
 [LoggingExternalInterface logging 2]
 [LoggingExternalInterface logging 42]
```

Comment:
Hmm, nice, the last result in `static-casts` was, before this PR, 0 that become 1 after opts - which was an RTT bug I guess. But after the change to the test it is the same after opts as it should be...

Reply:
We should make this a lit test to make the output easier to match to the input :grimacing: